stonesoup/simulator/tests/test_detections.py

```diff
@@ -77,17 +77,20 @@ def test_switch_detection_simulator(
         assert time == initial_state.timestamp + step * timestep
 
     test_detections = set()
+    test_clutter_detections = set()
     for step, (time, detections) in enumerate(test_detector):
         test_detections |= detections
```

Comment:
Would `set |= otherset` be clearer as `set.update(otherset)`? To me, the latter seems to indicate more what the line is meant to do. I realize this is opinion-based though, so proposed only as an option.